README.org

```diff
@@ -42,7 +50,7 @@
     (just put it somewhere in your PATH with the name ~opam~):
 
     #+BEGIN_SRC bash
-    wget https://github.com/ocaml/opam/releases/download/1.2.2/opam-1.2.2-x86_64-Darwin
+    wget https://github.com/ocaml/opam/releases/download/2.0.3/opam-2.0.3-x86_64-darwin
```

Comment:
2.0.4 or too soon?

Reply:
It's not there yet, but can go in this PR!